start.sh

```diff
@@ -35,11 +35,14 @@ fi
 ## Query Node Infrastructure
 ./query-node/start.sh
 
-## Orion
-./start-orion.sh
+if [ "${SKIP_ORION}" != true ]
+then
+  ## Orion
+  ./start-orion.sh
 
-## Storage Squid
-docker-compose -f ./docker-compose.storage-squid.yml up -d
```

Comment:
I don't think you mean to also prevent storage-squid from starting up if SKIP_ORION == true
The storage infra needs it to function, as they do not rely on query node anymore.